base_view_inheritance_extension/tests/test_base_view_inheritance_extension.py

```diff
@@ -104,7 +104,7 @@ def test_update_context_complex(self):
         ]
         self.assertEqual(
             res.xpath('//field[@name="invoice_line_ids"]')[0].attrib["context"],
-            "{%s}" % ", ".join(expected_items),
+            "{%s}" % ", ".join(expected_items),  # noqa: UP031
```

Comment:
This is easy enough to avoid needing and we should try as hard as possible to only have absolutely necessary linting exceptions IMO. Odoo's own CI now fails if a linting exception is used after some high profile security issues introduced. That said I am not sure the current state of f-string vs format vs printf discussions in OCA. Last I knew it was f-string bad, printf bad, use format. Maybe someone who knows better can help here as a few f-string conversions up higher.

Anyway it won't stop me installing and testing so will give it a go now.